packages/cacti-copm-core/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@hyperledger/cacti-copm-core",
+  "version": "0.0.1",
+  "description": "Cacti Common Operators",
+  "homepage": "https://github.com/hyperledger/cacti#readme",
+  "bugs": {
+    "url": "https://github.com/hyperledger/cacti/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperledger/cacti.git"
+  },
+  "license": "Apache-2.0",
+  "author": "Jennifer Bell",
+  "main": "dist/lib/main/typescript/index.js",
+  "module": "dist/lib/main/typescript/index.js",
+  "types": "dist/lib/main/typescript/index.d.ts",
+  "files": [
+    "dist/*"
+  ],
+  "scripts": {
+    "bundleopenapi": "yarn run --top-level tools:bundle-open-api-tpl-files",
+    "codegen": "yarn run --top-level run-s 'codegen:*'",
+    "codegen:proto": "yarn run proto:openapi; yarn run proto:connectrpc",
+    "dev:redo": "yarn run bundleopenapi; yarn run codegen:proto",
+    "proto:connectrpc": "yarn run buf generate --template=./buf.gen.yaml ./src/main/proto/generated/openapi",
+    "proto:openapi": "yarn run --top-level openapi-generator-cli generate -i ./src/main/json/openapi.json -g protobuf-schema --model-name-suffix=PB --language-specific-primitives=google.protobuf.Any --type-mappings=AnyType=google.protobuf.Any --type-mappings=object=google.protobuf.Any --additional-properties=packageName=org.hyperledger.cacti.plugin.cacti.plugin.copm.core -o ./src/main/proto/generated/openapi/ -t=./src/main/mustache/openapi-generator/templates/protobuf-schema/ --ignore-file-override ../../openapi-generator-ignore"
+  },
+  "dependencies": {
+    "@bufbuild/buf": "1.30.0",
+    "@bufbuild/protobuf": "1.10.0",
+    "@connectrpc/connect": "^1.4.0",
+    "@hyperledger/cacti-weaver-sdk-fabric": "workspace:^",
+    "@hyperledger/cactus-common": "workspace:^"
+  },
+  "devDependencies": {
+    "@bufbuild/protoc-gen-es": "1.8.0",
+    "@connectrpc/connect-node": "^1.4.0",
```

Comment:
@jenniferlianne Please make sure to pin the dependencies to exact versions everywhere.

Reply:
OK.